charts/bitwardenrs/README.md

```diff
@@ -64,7 +64,13 @@ helm install bitwardenrs k8s-at-home/bitwardenrs -f values.yaml
 
 ## Custom configuration
 
-N/A
+### Ldap-Sync
```

Comment:
This will get overridden, do not make the changes here in the README.md

You need to update `README_CONFIG.md.gotmpl` and then run `./hack/gen-helm-docs.sh stable bitwardenrs` in your shell.

Reply:
Sure thing I will add these changes to the appropriate file and rebase.

Reply:
Updated.